src/hooks/useLatestCallback.ts

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+
+const useIsomorphicLayoutEffect =
+  typeof window !== 'undefined' ? React.useLayoutEffect : React.useEffect;
```

Comment:
hey, we rather use `import { useEffect } from 'react`
etc. Not sure why here this is different?

Reply:
Following Terry's advice I copied it over from his [PR](https://github.com/rainbow-me/rainbow/pull/3547/files#diff-aa6a3ef706bb0a495227d6c59c5f491546ccf76a5772a4c735238376ab7696aeR92) so I'd rather discuss it there and re-sync here as this is intended to be exactly the same file post-merges.